source/common/upstream/logical_dns_cluster.cc

```diff
@@ -87,20 +83,9 @@ void LogicalDnsCluster::startResolve() {
               Network::Utility::getAddressWithPort(*address_list.front(),
                                                    Network::Utility::portFromTcpUrl(dns_url_));
           if (!logical_host_) {
-            // TODO(mattklein123): The logical host is only used in /clusters admin output. We used
-            // to show the friendly DNS name in that output, but currently there is no way to
-            // express a DNS name inside of an Address::Instance. For now this is OK but we might
-            // want to do better again later.
-            switch (address_list.front()->ip()->version()) {
```

Comment:
How come we don't care about the ip version anymore?

Reply:
It will come directly from the DNS resolver so we don't need this 0.0.0.0 hack anymore.